src/did/Did.ts

```diff
@@ -67,7 +68,14 @@ export interface IDidDocumentPublicKey {
 export interface IDidDocumentPpties {
   authentication: object
   publicKey: IDidDocumentPublicKey[]
-  service: any
+  service: IDidService[]
+}
+
+export interface IDidService {
+  type: string
+  serviceEndpoint: string | AnyJsonObject
+  id?: string
+  [key: string]: AnyJson
 }
```

Comment:
we implemented this on the basis of the [W3C draft](https://www.w3.org/TR/did-core/#service-endpoints), hope that makes sense to you @maudnals